lib/screens/overview_screen/widgets/input_card_section.dart

```diff
@@ -71,26 +72,20 @@ class InputCardSection extends StatelessWidget {
                 ],
               ),
               const SizedBox(height: 10),
-              IntrinsicWidth(
-                stepWidth: double.infinity,
-                child: ElevatedButton(
-                  onPressed: () {
-                    Navigator.push(
-                      context,
-                      MaterialPageRoute<Widget>(
-                        builder: (BuildContext context) =>
-                            const QuestionaireScreen(),
-                      ),
-                    );
-                  },
-                  style: ElevatedButton.styleFrom(
-                    backgroundColor: Colors.green,
-                  ),
-                  child: Text(
-                    AppLocalizations.of(context)!.howFeel,
-                  ),
-                ),
-              )
+              PicosInkWellButton(
+                padding: const EdgeInsets.symmetric(horizontal: 0),
+                text: AppLocalizations.of(context)!.howFeel,
+                onTap: () {
+                  Navigator.push(
```

Comment:
Please use the navigator properly with named routes.

Reply:
Done